compiler/quilt/tools/command.py

```diff
@@ -934,28 +947,39 @@ def install(package, hash=None, version=None, tag=None, force=False):
 
     print("Downloading package metadata...")
 
-    if version is not None:
-        response = session.get(
-            "{url}/api/version/{owner}/{pkg}/{version}".format(
-                url=get_registry_url(team),
-                owner=owner,
-                pkg=pkg,
-                version=version
+    try:
+        if version is not None:
+            response = session.get(
+                "{url}/api/version/{owner}/{pkg}/{version}".format(
+                    url=get_registry_url(team),
+                    owner=owner,
+                    pkg=pkg,
+                    version=version
+                )
             )
-        )
-        pkghash = response.json()['hash']
-    elif tag is not None:
-        response = session.get(
-            "{url}/api/tag/{owner}/{pkg}/{tag}".format(
-                url=get_registry_url(team),
-                owner=owner,
-                pkg=pkg,
-                tag=tag
+            pkghash = response.json()['hash']
+        elif tag is not None:
+            response = session.get(
+                "{url}/api/tag/{owner}/{pkg}/{tag}".format(
+                    url=get_registry_url(team),
+                    owner=owner,
+                    pkg=pkg,
+                    tag=tag
+                )
             )
-        )
-        pkghash = response.json()['hash']
-    else:
-        pkghash = _match_hash(session, team, owner, pkg, hash)
+            pkghash = response.json()['hash']
+        else:
+            pkghash = _match_hash(session, team, owner, pkg, hash)
+    except HTTPResponseException as e:
+        logged_in_team = _find_logged_in_team()
+        if team is None and logged_in_team is not None:
+            if e.response.status_code == 404:
```

Comment:
Should be part of the `if` above - otherwise, you're gonna drop the exception

Reply:
Also, you should use `requests.codes.not_found`, for consistency with other code. (In case, you know, someone doesn't know what a 404 is...)